### PR TITLE
Add Khmer language support

### PR DESCRIPTION
**Pull request by sovichet**
Hello. This is a commit of adding Khmer language (km) support. Please kindly merge this.
Thanks.

### CODE REVIEW

**Comment by lucasmichot** [comment]
Can you add tests too @sovichet ?

**Comment by sovichet** [comment]
@lucasmichot Yes, I will do it.

**Comment by sovichet** [comment]
@lucasmichot Test for Khmer added.

File: tests/Localization/KmTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+use Tests\AbstractTestCase;
+
+class KmTest extends AbstractTestCase
+{
+    public function testDiffForHumansUsingShortUnitsKhmer()
+    {
+        Carbon::setLocale('km');
+
+        $scope = $this;
+        $this->wrapWithTestNow(function () use ($scope) {
```
Comment (lucasmichot) [inline tests/Localization/KmTest.php]: This should be `$this->wrapWithNonDstDate`

File: src/Carbon/Lang/km.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/*
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
```
Comment (lucasmichot) [inline src/Carbon/Lang/km.php]:
```php
/**
 * Translation messages.  See http://symfony.com/doc/current/book/translation.html
 * for possible formats.
 */
```

This comment is useless and shall be removed

Comment (lucasmichot) [inline src/Carbon/Lang/km.php]: Ping @sovichet

**Comment by lucasmichot** [comment]
@briannesbitt I feel most translation-related PR are safe to be merged.

**Comment by briannesbitt** [comment]
ok thanks.

**Pull request merged**